includes/layout/functions.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Useful functions
+ *
+ * @since 1.0.0
```

Comment:
@vibhavsalelkar update the version number

Reply:
updated in new commit